### PR TITLE
Some PEP8 tweaks for new settings.

### PR DESCRIPTION
**Pull request by ajrbyers**

### CODE REVIEW

File: src/core/janeway_global_settings.py [hunk 1]
```diff
@@ -443,18 +443,23 @@ def filter(self, record):
     for t in TEMPLATES:
         t["OPTIONS"]["string_if_invalid"] = "Invalid variable: %s!!"
     MIDDLEWARE_CLASSES = (('utils.middleware.TimeMonitoring',)
-        + MIDDLEWARE_CLASSES)
+                          + MIDDLEWARE_CLASSES)
 
 # Testing Overrides
 if IN_TEST_RUNNER and COMMAND[1:2] != ["--keep-db"]:
     from collections.abc import Mapping
+
```
Comment (mauromsl) [inline src/core/janeway_global_settings.py]: If we are changing this, it should be two

File: src/core/janeway_global_settings.py [hunk 2]
```diff
@@ -443,18 +443,23 @@ def filter(self, record):
...
     class SkipMigrations(Mapping):
+
```
Comment (mauromsl) [inline src/core/janeway_global_settings.py]: Why this new line?

File: src/utils/middleware.py [hunk 3]
```diff
@@ -23,13 +23,16 @@ def process_response(self, request, response):
             del _local.request
         return response
 
+
 class TimeMonitoring(object):
     """Monitors the resource usage of a request/response cycle """
+
```
Comment (mauromsl) [inline src/utils/middleware.py]: Why this new line?

File: src/core/janeway_global_settings.py [hunk 4]
```diff
@@ -443,18 +443,23 @@ def filter(self, record):
     for t in TEMPLATES:
         t["OPTIONS"]["string_if_invalid"] = "Invalid variable: %s!!"
     MIDDLEWARE_CLASSES = (('utils.middleware.TimeMonitoring',)
-        + MIDDLEWARE_CLASSES)
+                          + MIDDLEWARE_CLASSES)
```
Comment (mauromsl) [inline src/core/janeway_global_settings.py]: indented by 26 spaces?

**Comment by ajrbyers** [comment]
New tweaks added.

**Pull request merged**